src/Adapters/AzureManagementAdapter.ts

```diff
@@ -99,9 +99,7 @@ export default class AzureManagementAdapter implements IAzureManagementAdapter {
             },
             data: isResourceGraphCall(params)
                 ? {
-                      query: `Resources | where type =~ '${
```

Comment:
does it make sense to roll this behind a local storage flag? They can test it out and it it works, we can remove the flag and send it for real? That would help to avoid any potential regression for other folks

Reply:
my preference is to check in as is, and send to dev.  i think the chance of this regressing others is low from what i can tell reading the docs (the filter is basically optional), and it means we can roll to prod that much faster without another code change.  plus it makes validating it for customers a lot simpler, not everyone is trying to crack the console open like we are haha